test/functional/api/test_auth.py

```diff
@@ -0,0 +1,58 @@
+import pytest
+
+from httpx import AsyncClient
+
+from cli.app.commands.auth import generate_token
```

Comment:
let's not call cli commands
better to move it's logic to some function & call it in cli